### DISCUSSION TITLE
Ventilated cover to 3d print

### DISCUSSION THREAD

**Discussion by larroy** [OpenEVSE/open_evse#31] [Show and tell]
Hello I made a ventilated cover to 3d print. I noticed with the closed cover things got toasty inside charging at 48 amp. Probably  because of the ring terminals? not sure as I don't have an IR camera. https://www.printables.com/model/540063-openevse-ventilated-cover

**Comment by chris1howell** [comment]
We do not recommend charging open air. The sealed enclosure design addresses all three sides of the fire triangle, oxygen, heat and fuel.

Oxygen - if a fire were to start all oxygen is consumed immediately in a sealed enclosure, self extinguishing fire.
Heat - Station reduced current at a very conservative 65C internal temperature and shuts down at 72C.
Fuel - Materials are all fire-resistant, high voltage wire insulation is 105C.

Ventilation provides endless oxygen and the relay temperature could run away without detection of the temperature sensor.

Most of the heat is the resistance in the relay contacts and connectors, more current = more heat. Here is a support article on the subject.
https://openev.freshdesk.com/a/solutions/articles/6000248724

**Comment by larroy** [comment]
My OpenEVSE temperature sensor stopped working and there's absolutely zero response from support.
